### DISCUSSION TITLE
it didn't work

### DISCUSSION THREAD

**Discussion by mjeed02** [Inpacchi/Manga-Tagger#45] [Q&A]
hi
i did everything in the installation & configuration
and in the end it says "Manga Tagger will automatically process and move them to the default library location of C:\Library"
but the "C:\Library" did not appear so the manga tagger didn't do anything

so can anybody help me
thanks

**Comment by mckenna654** [comment]
I think the app is dead. I've configured it to a t and it continues to fail to run
